cgnsutilities/cgnsutilities.py

```diff
@@ -924,8 +841,6 @@ def tanDist(Sp1, Sp2, N):
             # Sp2: final spacing (within the [0,1] interval)
             # N: number of nodes
 
-            # IMPORTS
-            from numpy import tan, arange, pi
             from scipy.optimize import minimize
```

Comment:
Why did we move `import re` outside, but not `import scipy`

Reply:
Keeping the SciPy import inside the function means that the user does not need to have SciPy if they are not using the function. This is directly related to keeping it as an 'advanced' dependency.

I added the 'advanced' features because I didn't want to add pySpline as a hard dependency for a single function (rebunch) that I'm pretty sure no one uses. I think adding SciPy as a hard dependency is justifiable though if we want to do that because it doesn't require compiling another MDO Lab code.

Reply:
Yeah, I was thinking something along these lines. SciPy is a pretty common package.